Cubical/Categories/DistLatticeSheaf/ComparisonLemma.agda

```diff
@@ -0,0 +1,445 @@
+{-# OPTIONS --safe --lossy-unification #-}
```

Comment:
Add a reference to https://ncatlab.org/nlab/show/comparison+lemma or similar to explain the name a bit?